sdk/spring/azure-spring-boot-samples/azure-spring-data-sample-gremlin-web-service/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-spring-boot-samples</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>azure-spring-data-sample-gremlin-web-service</artifactId>
+  <name>Spring Data gremlin - Web Service</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-autoconfigure</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.jayway.jsonpath</groupId>
+      <artifactId>json-path</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.microsoft.spring.data.gremlin</groupId>
+      <artifactId>spring-data-gremlin</artifactId>
+      <version>2.3.1-beta.1</version>
```

Comment:
Does the update-version script take care of this scenario now?

Reply:
@saragluna the update script isn't the problem, it's the verification scripts. If this isn't tagged and dependencies are updated it'll get skipped.

Reply:
Updated. So will the verification scripts scan these dependencies defined in spring samples too? Given that they're not parenting off the sdk-client-parent pom, but using spring-boot-parent.

Reply:
@saragluna no. The [update_versions](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/update_versions.py) script will still update the tag, it's the [scanner script](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/pom_file_version_scanner.ps1) that doesn't verify because of the parentage. This has allowed several tags that were either incorrect or had incorrect versions to get checked in.